ViewModel/TableWithPlot/Rows/GraphicalRow.cs

```diff
@@ -10,7 +10,37 @@ public class GraphicalRow : SequentialRow
     {
         private double _Confidence95 = 0;
         private double _Confidence05 = 0;
+        private double _x;
 
+        [DisplayAsColumn("X Value")]
+        public override double X
+        {
+            get { return _x; }
+            set
+            {
+                _x = value;
+                NotifyPropertyChanged();
+                ((GraphicalRow)PreviousRow)?.NotifyPropertyChanged(nameof(X));
+                ((GraphicalRow)NextRow)?.NotifyPropertyChanged(nameof(X));
+            }
+
+        }
+        [DisplayAsColumn("Y Value")]
+        [DisplayAsLine("Y Value")]
+        public double Value
+        {
+            get
+            {
+                return ((Deterministic)Y).Value;
+            }
+            set
+            {
+                Y = new Deterministic(value);
+                Confidence05 = 0;
+                Confidence95 = 0;
+                NotifyPropertyChanged();
+            }
+        }
         [DisplayAsColumn("5% Confidence")]
```

Comment:
I know this is not new code, but would just like to mention that the field is used to a 95% confidence interval. If we're trying to maintain as smooth of a transition as practicable, then I think a 95% interval would be better.

Reply:
Easily done. Right now this class isn't linked to a view model. I'll make sure that lines up when I move that in.